trunk/src/app/srs_app_rtc_source.hpp

```diff
@@ -335,6 +342,13 @@ class SrsRtcFrameBuilder
     uint16_t header_sn_;
     uint16_t lost_sn_;
     int64_t rtp_key_frame_ts_;
+
+    // Audio jitter buffer, map sequence number to packet
+    std::map<uint16_t, SrsRtpPacket*> audio_buffer_;
```

Comment:
Obviously, the audio jitter buffer should be a class with its own fields and behaviors, rather than being directly exposed and mixed with the SrsRtcFrameBuilder code.

```
    // Audio jitter buffer, map sequence number to packet
    std::map<uint16_t, SrsRtpPacket*> audio_buffer_;
    // Last processed sequence number
    uint16_t last_audio_seq_num_;
    // Last time we processed the jitter buffer
    int64_t last_audio_process_time_ms_;
```